index.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<html class="no-js" lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Home Task 2</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="css/index.css">
+</head>
+<body>
+<header class="NavBar">
+  <div class="NavContainer">
+    <div class="Logo">
+      <h5 class="logoitem">Company name</h5>
+    </div>
+    <div class="NavList">
+      <ul class="NavListitems">
+        <li><a class="NavItem" href="">Pricing</a></li>
+        <li><a class="NavItem" href="">Support</a></li>
+        <li><a class="NavItem" href="">Enterprise</a></li>
+        <li><a class="NavItem" href="">Features</a></li>
+      </ul>
+    </div>
+    <div class="Button">
+      <button class="SignButton" type="button">Sign up</button>
+    </div>
+  </div>
+</header>
+<main class="MainContent">
+  <section>
+    <div class="SectionContainer1">
```

Comment:
what the difference between `SectionContainer1 `, `SectionContainer2`, `SectionContainer3`

can you have only one? e.g. `section`?